src/blog/2022/12/Create-http-trigger-with-authentication.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Create HTTP triggers with authentication
+subtitle: From any browser or command line you now have the ability to securely trigger your flows
+description: From any browser or command line you now have the ability to securely trigger your flows
+date: 2022-12-07
+authors: ["rob-marcer"]
+---
+
+
+Having an HTTP endpoint trigger your flows is very useful. From any browser or command line you now have the ability to trigger your flows.
+<!--more-->
+Doing so safely with authentication is slightly harder, but not a lot. FlowForge makes it rather easy to accomplish.
+
+### Creating the HTTP flow
+
+When you start a project on FlowForge, remember the project name. For this how-to we’ll use `example`. Open the editor and drag in the HTTP In node as well as the HTTP response node. Connect them, and add a debug node, which is connected to the “HTTP in” node.
+
+First off; let’s set the HTTP in node properties:
+
+![Shows the UI to edit the node's properties](../images/edit-http-node.png)
+
+You can import this flow into your own project if you’d like:
+
+```
```

Comment:
Can we add this in a details block?

```
<details>
....
</details>
```

I bet it folds the JSON away, and doesn't intrude too much